assets/js/admin-scripts.js

```diff
@@ -1212,12 +1212,17 @@ jQuery(document).ready(function ($) {
 			$('body').on( 'submit', '.edd-export-form', function(e) {
 				e.preventDefault();
 
-				var data = $(this).serialize();
+				if ( ! $(this).find( 'input[type="submit"]' ).hasClass( 'button-disabled' ) ) {
 
```

Comment:
We should cache this selector for the button, since we use it twice:
```
var submitButton = $(this).find( 'input[type="submit"]' );
if ( ! submitButton.hasClass( 'button-disabled' ) ) {

	var data = $(this).serialize();

	submitButton.addClass( 'button-disabled' );
```